python/lsst/ts/m2gui/controltab/tab_actuator_control.py

```diff
@@ -33,7 +57,516 @@ class TabActuatorControl(TabDefault):
+            if actuator_selection.isChecked():
+                actuators.append(idx)
+
+        target_displacement = self._target_displacement.value()
+
+        # Index begins from 0 instead of 1 in QComboBox
+        displacement_unit = ActuatorDisplacementUnit(
+            self._displacement_unit_selection.currentIndex() + 1
+        )
+
+        run_command(
+            self.model.command_actuator,
+            CommandActuator.Start,
+            actuators,
+            target_displacement,
+            displacement_unit,
+        )
+
+        return actuators, target_displacement, displacement_unit
+
+    @Slot()
+    def _callback_actuator_command(self, command):
+        """Callback of the actuator buttons in actuator control. The cell
+        controller will run the related command.
+
+        Parameters
+        ----------
+        command : enum `CommandActuator`
+            Actuator command.
+        """
+        run_command(self.model.command_actuator, command)
+
+    def _set_widget_and_layout(self):
+        """Set the widget and layout."""
+
+        widget = self.widget()
+        widget.setLayout(self._create_layout())
+
+        # Resize the dock to have a similar size of layout
+        self.resize(widget.sizeHint())
+
+        self.setWidget(self.set_widget_scrollable(widget, is_resizable=True))
+
+    def _create_layout(self):
+        """Create the layout.
+
+        Returns
+        -------
+        layout : `PySide2.QtWidgets.QHBoxLayout`
+            Layout.
+        """
+
+        layout = QHBoxLayout()
+
+        # First column
+        layout_script_control_force_summary = QVBoxLayout()
+        layout_script_control_force_summary.addWidget(
+            self._create_group_script_control()
+        )
+        layout_script_control_force_summary.addWidget(
+            self._create_group_force_summary()
+        )
+
+        layout.addLayout(layout_script_control_force_summary)
+
+        # Second column
+        layout_actuator_control = QVBoxLayout()
+        layout_actuator_control.addWidget(self._create_group_actuator_selector())
+        layout_actuator_control.addWidget(self._create_group_actuator_control())
+
+        layout.addLayout(layout_actuator_control)
+
+        return layout
+
+    def _create_group_script_control(self):
+        """Create the group of script control. The cell control system reads
+        the binary script to perform a seris of actuator motions.
+
+        Returns
+        -------
+        group : `PySide2.QtWidgets.QGroupBox`
+            Group.
+        """
+
+        layout = QVBoxLayout()
+
+        layout_file = QFormLayout()
+        layout_file.addRow("Script File:", self._info_script["file"])
+        layout_file.addRow("Progress:", self._info_script["progress"])
+
+        layout.addLayout(layout_file)
+
+        layout_control = QGridLayout()
+        layout_control.addWidget(self._buttons_script["load_script"], 0, 0)
```

Comment:
See above. Organizing things in maps makes sense when you need that for e.g. value setting. You aren't setting buttons. Either use common class variables (self.btn_load_script, etc..) or see above how that can be optimized.